### ISSUE TITLE
verify phy to FMAT conversion scripts

### ISSUE THREAD

**Issue by DavidTingley** [buzsakilab/buzcode#3]

**Comment by brendonw1** [comment]
Do these "verifys" mean that these things have been verified as of this
push?

**Comment by DavidTingley** [comment]
No, it's more of a "to do" list.  I'll close each of these issues once I've verified that things are working and (semi) well commented
